eco.Rmd

```diff
@@ -25,42 +25,47 @@ finally, point to books on ecological modeling especially emphasizing Alain Zuur
-The desert's extreme harsh conditions and its remoteness spawned a truly unique and endemic flora and fauna on these mountains.
-The annual rain input is less than 30-50 mm on average.
-Still, plants survive this extreme aridity by combing out fog which develops below the temperature inversion caused by the cold Humboldt current in austral winter.
-Every once in a few years, El Ni{\~n}o brings torrential rainfalls to this sun-baked environment causing on the one hand the desert to bloom, and on the other hand giving tree seedlings the chance to develop roots long enough to survive the following aridity.
-Unfortunately, fog oases are heavily endangered mostly by human activities (grazing and agriculture) but also by climate change.
-To effectively protect the last remnants of this unique vegetation formation, we need to know its floristic composition and spatial distribution.
-*Lomas* mountains also have a touristical, i.e. economical value.
-The vast majority of Peruvians are living in the coastal desert, and seldomly have a chance to leave it.
-Scientists long found out that green oases are of utmost importance for human well-being, especially for metropolitan dwellers such as the people from Lima.
-The only green space in the immediate vicinity of Lima are *lomas* mountains.
-This even increases the immense recreational value of such green oases, and the need to effectively protect them.
-
-In this chapter we would like to analyze the floristic composition of the southen slope of Mt. Mongón, a *lomas* mountain near Casma on the central northern coast of Peru.
+The desert's extreme conditions and remoteness provide the habitat for a unique ecosystem, including species endemic to the fog oases.
+Despite the arid conditions and low levels of precipitation of around 30-50 mm per year, plants can survive, by 'combing out' fog.
+This fog, which develops below the temperature inversion caused by the cold Humboldt current in austral winter, provides the name for this habitat.
+Every few years, the El Niño weather pattern brings torrential rainfall to this sun-baked environment.
+This causes the desert to bloom, and provides tree seedlings a chance to develop roots long enough to survive the arid conditions until the next rainfall.
+
+Unfortunately fog oases are heavily endangered.
+This is mostly due to human activity (agriculture and climate change).
+To effectively protect the last remnants of this unique vegetation ecosystem, evidence is needed on the composition and spatial distribution of the native flora.
+*Lomas* mountains have economic value as a tourist destination (although most Peruvians live in the coastal desert, and seldomly have the opportunity to visit them) and can contribute to the wellbeing of local people via recreation.
+
+In this chapter we will demonstrate ecological applications of some of the techniques learned in the previous chpaters.
```

Comment:
Context. Will be good to cross-ref these at some point but no rush.

Reply:
Yes, will do so.